src/app/subcategory/[subcategoryId]/SubCategoryItems.tsx

```diff
@@ -0,0 +1,35 @@
+import { ItemsForm } from '../components/itemsForm';
+import { ListTitle } from '../components/ListTitle';
+import type { ListProps } from '../types';
+import { getData } from '../utils/getData';
+import classes from './styles.module.css';
+
+const List = async ({ params }: ListProps) => {
+  const subcategoryId = params.subcategoryId;
+  const { subcategoryItems, error, message } = await getData(
+    'subcategoryItems',
+    subcategoryId
+  );
+  const {
+    subcategory,
+    error: subategoryError,
+    message: subcategoryMessage,
+  } = await getData('subcategory', subcategoryId);
+
+  return (
+    <main className={classes.main}>
+      {subcategory && <ListTitle list={subcategory} />}
+      {subcategoryId && (
+        <>
+          <ItemsForm
+            listOfItems={subcategoryItems}
+            subcategoryId={subcategoryId}
+          />
+          <p>{(error || subategoryError) && (message || subcategoryMessage)}</p>
```

Comment:
If we have both type of errors? 
Should we add special style for errors? 
For example, add class .error-text {color: var(--color-primary-pink)}